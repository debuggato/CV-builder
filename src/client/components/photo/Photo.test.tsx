import React from 'react';
import renderer from 'react-test-renderer';
import PhotoView from './Photo';

describe('', () => {
	it('expects to render photo component WITH photo url set', () => {

		const onUpload = () => { }

		const wrapper = renderer
			.create(
				<PhotoView onUpload={onUpload} imgUrl="/assets/photo_profile.jpg" />
			)
			.toJSON();

		expect(wrapper).toMatchSnapshot();
	});
});