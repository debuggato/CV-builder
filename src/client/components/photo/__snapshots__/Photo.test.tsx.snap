// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render photo component WITH photo url set 1`] = `
<div
  className="sc-AxiKw hwFSgf"
>
  <img
    src="/assets/photo_profile.jpg"
    style={
      Object {
        "marginRight": "10px",
        "width": "100px",
      }
    }
  />
  <label
    htmlFor="upload"
  >
    <span
      className="sc-AxhCb hgARnT"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-upload fa-w-16 "
        data-icon="upload"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={
          Object {
            "marginRight": "10px",
          }
        }
        viewBox="0 0 512 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M296 384h-80c-13.3 0-24-10.7-24-24V192h-87.7c-17.8 0-26.7-21.5-14.1-34.1L242.3 5.7c7.5-7.5 19.8-7.5 27.3 0l152.2 152.2c12.6 12.6 3.7 34.1-14.1 34.1H320v168c0 13.3-10.7 24-24 24zm216-8v112c0 13.3-10.7 24-24 24H24c-13.3 0-24-10.7-24-24V376c0-13.3 10.7-24 24-24h136v8c0 30.9 25.1 56 56 56h80c30.9 0 56-25.1 56-56v-8h136c13.3 0 24 10.7 24 24zm-124 88c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20zm64 0c0-11-9-20-20-20s-20 9-20 20 9 20 20 20 20-9 20-20z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      upload.photo
    </span>
    <input
      className="sc-AxjAm bSkJNt"
      id="upload"
      onChange={[Function]}
      type="file"
    />
  </label>
</div>
`;
